### PR TITLE
Add plugin hexo-filter-inline-svg

### PR DESCRIPTION
**Pull request by WiderGao**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [x] I want to publish my plugin on Hexo official website.
  - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [x] `name` is unique.
  - [x] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English
  - [ ] `ko` Korean
  - [ ] `pt-br` Brazilian Portuguese
  - [ ] `ru` Russian
  - [ ] `th` Thai
  - [ ] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

File: source/_data/plugins.yml [hunk 1]
```diff
@@ -3034,3 +3034,10 @@
     - javascript
     - js
     - css
+- name: hexo-filter-inline-svg
+  description: A Hexo plugin that inline your svg in the img tag into HTML file.
+  link: https://github.com/widergao/hexo-filter-inline-svg
+  tags:
+    - filter
+    - inline
+    - svg
```
Comment (jiangtj) [inline source/_data/plugins.yml:3043]: Could you leave a blank line at the end of file?

Comment (WiderGao) [reply source/_data/plugins.yml]: OK, how about now.

Comment (jiangtj) [reply source/_data/plugins.yml]: It looks like you added two lines 😂

Comment (jiangtj) [reply source/_data/plugins.yml]: https://eslint.org/docs/rules/eol-last

Comment (WiderGao) [reply source/_data/plugins.yml]: I'm sorry, is it right this time? I used the online editor in github but I don't know why the number of lines is not the same as the real one.

**Review by jiangtj — APPROVED** [review]
👍

**Review by yoshinorin — APPROVED** [review]

**Pull request merged**